test/powershell/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.Tests.ps1

```diff
@@ -0,0 +1,501 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Function Install-ModuleIfMissing
+{
+    param(
+        [parameter(Mandatory)]
+        [String]
+        $Name,
+        [version]
+        $MinimumVersion,
+        [switch]
+        $SkipPublisherCheck,
+        [switch]
+        $Force
+    )
+
+    $module = Get-Module -Name $Name -ListAvailable -ErrorAction Ignore | Sort-Object -Property Version -Descending | Select-Object -First 1
+
+    if(!$module -or $module.Version -lt $MinimumVersion)
+    {
+        Write-Verbose "Installing module '$Name' ..." -Verbose
+        Install-Module -Name $Name -Force -SkipPublisherCheck:$SkipPublisherCheck.IsPresent
+    }
+}
+
+Function Test-IsInvokeDscResourceEnable
+{
+    return [ExperimentalFeature]::IsEnabled("PSDesiredStateConfiguration.InvokeDscResource")
+}
+
+Describe "Test PSDesiredStateConfiguration" -tags CI {
+    Context "Module loading" {
+        BeforeAll {
+            $commands = Get-Command -Module PSDesiredStateConfiguration
+            $expectedCommandCount = 3
+            if (Test-IsInvokeDscResourceEnable)
+            {
+                $expectedCommandCount ++
+            }
+        }
+        BeforeEach {
+        }
+        AfterEach {
+        }
+        AfterAll {
+        }
+
+        It "The module should have $expectedCommandCount commands" {
+            if($commands.Count -ne $expectedCommandCount)
+            {
+                $modulePath = (Get-Module PSDesiredStateConfiguration).Path
+                Write-Verbose -Verbose -Message "PSDesiredStateConfiguration Path: $modulePath"
+                $commands | Out-String | Write-Verbose -Verbose
+            }
+            $commands.Count | Should -Be $expectedCommandCount
+        }
+        It "The module should have the Configuration Command" {
```

Comment:
Please add whitespace between It statements for readability

Reply:
```suggestion

        It "The module should have the Configuration Command" {
```